pom.xml

```diff
@@ -91,9 +91,28 @@
 					<groupId>logback-classic</groupId>
 					<artifactId>slf4j-log4j12</artifactId>
 				</exclusion>
+				<exclusion>
+					<groupId>ch.qos.logback</groupId>
+					<artifactId>logback-classic</artifactId>
+				</exclusion>
```

Comment:
You probably shall exclude logback-classic from spring-boot-starter-logging artifact?

Reply:
It's excluded from spring-boot-starter-logging from web-starter.
<img width="650" alt="Screenshot 2023-12-04 at 2 11 51 PM" src="https://github.com/eclipse-tractusx/managed-simple-data-exchanger-backend/assets/91305578/177dd4a9-796e-4095-8d70-3fea622a3d24">

Reply:
As I see, spring-boot-starter-logging is a transitive dependency for web-starter, so I approve